### ISSUE TITLE
ProjectM will not start unless using MS store version

### ISSUE THREAD

**Issue by ghost** [projectM-visualizer/projectm#423]
Hi there, 
ProjectM will start if I install the MS Store version, but it will not start when I try the Standalone or Steam version.

I first installed the MS store version, and was able to open ProjectM.
I uninstalled it, then downloaded the standalone version and executed the exe.
It ran the first time, but after exiting and re-executing the exe, ProjectM has not run again since. 
I installed the Steam version and it seemed to be executing something, but the ProjectM window does not open. 

When I check the task manager, there is nothing to indicate ProjectM is running or struggling to run.

Any thoughts on a fix for this?

Note: I can not use the MS store version for what I intend on using ProjectM for due to limitations on UWP applications.
Thanks so much!
Kyle

**Comment by chaosophy** [comment]
I 'think' - been a while since I looked tbh - that all versions might use the same config file.

Look in your c:\users\{username} folder for a folder called .projectM (notice the name starts with a '.')
Folder should be in there, if not it might be buried in appdata somewhere.

Might be worth a go to delete that?  Only thing I can think of that each version might be sharing and may cause an issue.

**Comment by aprice** [comment]
Same issue, and I can't find a .projectM folder anywhere under my user folder. The Windows Store version runs fine and picks up audio from other apps, but it doesn't have the preset browser/playlist, which I really miss from the Linux version.

I tried the Steam version to see if it was any better, it installs but doesn't run; click Play and it tries to launch but immediately closes, I get no application window, and Steam sees it as having run for 1 second and exiting. I can't find a log anywhere.

Grabbed the standalone Windows build, installed the latest VC++ redist, run it, nothing seems to happen at all. Ran from console to see if there was any stdout, but I get nothing.

Neither seems to produce any meaningful entries in WEL or any log file on disk.

**Comment by terolaakso** [comment]
I experienced the exact same issue. Fixed it by unchecking "Enable audio enhancements" in Advanded tab of Speaker Properties.

In my case the error seems to be with Nahimic audio enhancements. I debugged the latest source, and then an error is output: `avcore\audiocore\client\audioclient\audioclientcore.cpp(1485)\AudioSes.dll!00007FFDCF494DCB: (caller: 00007FFDCF47288D) ReturnHr(1) tid(4b48) 887C0030`. `pAudioClient` comes from ASM.Plugin.AudioDevProps2.dll which is part of Nahimic.
